app/components/bestuursperiode-new-button.hbs

```diff
@@ -2,6 +2,7 @@
   <AuButton {{on "click" this.openModal}}>
```

Comment:
Is it possible that this component is not used anymore?

Reply:
I can't seem to find an implementation. Will remove it in a separate PR